test/unit/push_gateway_test.exs

```diff
@@ -2,59 +2,68 @@ defmodule PushGatewayTest do
   @moduledoc false
 
   use ExUnit.Case
-  import ExUnit.CaptureLog
+  use Placebo
+
+  import SmartCity.Event,
+    only: [
+      data_ingest_start: 0
+    ]
+
+  alias SmartCity.TestDataGenerator, as: TDG
+  import SmartCity.TestHelper, only: [eventually: 3]
+
+  @instance :push_gateway
+  @topic_prefix Application.get_env(:push_gateway, :topic_prefix)
+  @assigned_dataset_id Application.get_env(:push_gateway, :assigned_dataset_id)
+
+  @test_message_one DSRCMessages.bsm_message()
```

Comment:
@test_bsm_message
and
@test_srm_message ?

Reply:
Sure!